components/block_gen.md

```diff
@@ -0,0 +1,102 @@
+# Block Generation
+
+The logic which is used to generate the rollup chain from L1.
+
+## Glossary
+
+- **Block Inputs**: All information in an Ethereum block required to generate the full block contents.
+- **Rollup Blockchain**: The rollup blocks generated by performing the rollup's state transition function over all block inputs.
```

Comment:
```suggestion
- **Rollup Blockchain**: The rollup blocks generated by performing the rollup's state transition function over all block inputs. For example, DOES include a state root for that block's input.
```

Reply:
could be worth just specifying this explicitly, what is NOT in block input that IS in yellow paper section `4.3. The Block`